samples/Mvc.RestrictiveFileUpload/RequestFormSizeLimitAttribute.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Mvc.RestrictiveFileUpload
+{
+    /// <summary>
+    /// Filter to set size limits for request form data
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public class RequestFormSizeLimitAttribute : Attribute, IAuthorizationFilter, IOrderedFilter
+    {
+        private readonly FormOptions _formOptions;
+
+        public RequestFormSizeLimitAttribute(long bufferBodyLengthLimitFormSizeInBytes, int order = 0)
+        {
+            _formOptions = new FormOptions()
+            {
+                BufferBodyLengthLimit = bufferBodyLengthLimitFormSizeInBytes
```

Comment:
Don't invent new names. If you don't like the existing names file a bug against the original.